tests/test_bayweapons.py

```diff
@@ -35,3 +35,8 @@ def test_add_remove():
     ship.remove_bayweapon(wep)
     assert ship.cost_total == 2
     assert ship.cargo == 100
+
+    # error checking for removing a non-existant bayweapon
+    ship.remove_bayweapon(wep)
+    assert ship.cost_total == 2
+    assert ship.cargo == 100
```

Comment:
Are you aware of the lack of newline

Reply:
Lotta stuff going on, I was not. Will change.